packages/odyssey-react-mui/src/components/ThemeProvider/ThemeProvider.tsx

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles";
+import { ReactElement } from "react";
+
+import { odysseyTheme } from "../../themes";
```

Comment:
Yes. The only crutch would be that you cannot style any Odyssey MUI components. I thought of adding a way to pass our `ThemeProvider` (wrapper) a MUI `theme`, but that didn't make sense. If it's in Odyssey, we should have it styled correctly. Also you, as a consumer, shouldn't even know we're using MUI.

Reply:
I guess I was thinking in the short-term as we get to a stable Odyssey, us (SIW) consumers might need to revert to using the MuiThemeProvider directly as we do now and pass the Odyssey theme with patches in (basically what this wrapper provides). I realize this isn't the intended API but it's a crutch in the event we need something more urgently than Odyssey can provide.

Reply:
My bad. Also, you guys need `MuiThemeProvider` directly anyway because you need to allow custom theming of Odyssey right?

I add a re-exported `ThemeProvider as MuiThemeProvider`.

Reply:
Keep in mind, you can always pull down MUI and still use `odysseyTheme` directly if you need.

Reply:
Yeah correct we will need to use the `odysseyTheme` directly in order to support theming

Reply:
Having the re-exported `MuiThemeProvider` is good so we won't need to install the matching MUI version independently.

This does have me thinking about if odyssey should expose the theme directly on its own `ThemeProvider` somehow since we're supposed to try to hide the MUI backing. With the current approach anyone that wants to theme Odyssey will have to break encapsulation.